CHANGELOG.md

```diff
@@ -44,6 +44,14 @@ Documentation for rocRAND is available at
 
 ### Changes
 
+* For device-side generators, you can now wrap calls to rocrand_generate_* inside of a hipGraph. There are a few 
+  things to be aware of:
+  - Generator creation (rocrand_create_generator), initialization (rocrand_initialize_generator), and destruction (rocrand_destroy_generator) must still happen outside the hipGraph.
```

Comment:
Might be useful to detect stream captures with `hipStreamGetCaptureInfo_v2` and throw an appropriate error when it happens. But this is non blocking IMHO.

Reply:
Thanks @Naraenda, that's a really good idea. Given how close we are to the deadline for 6.2, maybe I'll defer this to 6.3 if that's ok. I'll make a note of it.

Reply:
Thanks @Naraenda, that's a really good idea. Given how close we are to the deadline for 6.2, maybe I'll defer this to 6.3 if that's ok. I'll make a note of it.